Source/WebCore/rendering/svg/legacy/LegacyRenderSVGRoot.h

```diff
@@ -97,6 +97,8 @@ class LegacyRenderSVGRoot final : public RenderReplaced {
 
     LayoutRect localClippedOverflowRect(RepaintRectCalculation) const;
 
+    LayoutRect computeContentsRepaintOverflow() const;
```

Comment:
if this is ink overflow we should name it accordingly.

Reply:
@alanbaradlay - Thanks for sharing your knowledge. Learnt something new and have now updated and renamed PR title and function name (of newly introduced one).